NeuralNet.js

```diff
@@ -0,0 +1,171 @@
+class NeuralNet {
+    constructor(options) {
```

Comment:
(This is really a comment re: https://github.com/ml5js/ml5-library/pull/485/ but putting here in terms of thinking this all through.) For simple cases, I'm thinking about supporting passing in just number of inputs, number of outputs:

```javascript
let neuralnet = new NeuralNet(3, 2);
```

Reply:
I like making that explicit as well. We can do something similar to how we handle the other examples: e.g. `inputsOrOptions, outputsOrOptions`